pkg/cmd/openshift-kube-apiserver/kubeadmission/register.go

```diff
@@ -95,6 +95,8 @@ func NewDefaultOffPluginsFunc(kubeDefaultOffAdmission sets.String) func() sets.S
 		kubeOff.Delete(additionalDefaultOnPlugins.List()...)
 		kubeOff.Delete(openshiftAdmissionPluginsForKube...)
 		kubeOff.Delete(customresourcevalidationregistration.AllCustomResourceValidators...)
+		// TODO: Why is this necessary?
+		kubeOff.Insert("authorization.openshift.io/RestrictSubjectBindings")
```

Comment:
@stlaz adding this back in (didn't realize it was removed in the RBR->CRD previous PRs).  Why this is necessary I do not know, but when added things work.  Must be a good explanation but I'm at a loss.  (Why is this necessary for RBRs but not SCCs or CRQs?) :woman_shrugging:

Reply:
If you add this back, you must also have an e2e test with RBR to confirm that is working at all.

Reply:
I did not add it back, but I added e2e anyways : )